### ISSUE TITLE
Is there any limitation about the length of a video?

### ISSUE THREAD

**Issue by gll89** [kenshohara/3D-ResNets-PyTorch#147]
Hi everybody,

When seeing the codes, I did not find any limitation about the length of a video in the training set.  If that, the dimension along the frames of the input to ResNet is variable. How does this make sense?

**Comment by gll89** [comment]
I got the answer from the code. It has a function of temporal_transforms, which do padding.

**Comment by kaiqiangh** [comment]
Hi, I have a question about video frames. If I use pre-trained model with 64 frame segment (ResNeXt-101-64f-kinectics.pth), can I extract features for the videos that have the number of frames less than 64? Thanks
